src/components/presentational/DumbbellChart/DumbbellChart.css

```diff
@@ -0,0 +1,103 @@
+.mdhui-dumbbell-visualization {
+    padding-top: 25px;
+    padding-bottom: 25px;
+    height: 100%;
+}
+
+.mdhui-dumbbell-chart{
+    height: 200px;
+    width: 100%;
+    position: relative;
+}
+
+.mdhui-dumbbell-y-axis{
+    position: absolute;
+    flex-flow: nowrap;
+    display: flex;
+    flex-direction: column;
+    justify-content:flex-end;
+    width: 100%;
+    border-top-width: 1px;
+    border-top-style: solid;
+    border-top-color: #E7EFFF;
+}
+
+.mdhui-dumbbell-x-axis{
+    position: absolute;
+    bottom: -20px;
+}
+
+.mdhui-dumbbell-axis-text{
+    color: #BFBFBF;
+    text-align: center;
+    font-family: 'Inter';
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+    display: flex;
+    justify-content: right;
```

Comment:
In the design there is some space between the right side:
<img width="75" alt="Screenshot 2023-11-27 at 4 45 48 PM" src="https://github.com/CareEvolution/MyDataHelpsUI/assets/1643171/8a5b0857-cc8b-44ac-9582-10de50c968f6">

That is missing here:
<img width="33" alt="Screenshot 2023-11-27 at 4 45 00 PM" src="https://github.com/CareEvolution/MyDataHelpsUI/assets/1643171/5392707e-4e4e-4067-9b53-2eb3264ef329">